akka-docs/src/main/paradox/typed/coexisting.md

```diff
@@ -115,6 +115,26 @@ Scala
 Java
 :  @@snip [TypedWatchingUntypedTest.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/coexistence/TypedWatchingUntypedTest.java) { #typed }
 
-There is one caveat regarding supervision of untyped child from typed parent. If the child throws an exception we would expect it to be restarted, but supervision in Akka Typed defaults to stopping the child in case it fails. The restarting facilities in Akka Typed will not work with untyped children. However, the workaround is to add another untyped actor that takes care of the supervision, i.e. restarts in case of failure if that is the desired behavior.
+## Supervision
+
+The default supervision for untyped is to restart, this changed to stop for Akka Typed.
+
+### Untyped supervising Typed 
+
+When adapting an untyped `ActorContext` or `ActorSystem` to `spawn` a typed actor the parent of the new actor will an untyped actor. This means that
+the typed actor will have a default supervision policy to restart rather than stop. For example the following actor will restart if it fails:
+
+Scala
+:  @@snip [DefaultUntypedSupervisingTyped.scala](/akka-actor-typed-tests/src/test/scala/docs/akka/typed/coexistence/DefaultUntypedSupervisingTypedSpec.scala) { #spawn-untyped }
+
+Java
+:  @@snip [DefaultUntypedSupervisingTypedTest.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/coexistence/DefaultUntypedSupervisingTypedTest.java) { #spawn-untyped }
+
+### Typed supervising untyped
+
+The opposite is true for typed supervising untyped. If the child throws an exception we would expect it to be restarted, 
+but supervision in typed defaults to stopping the child in case it fails. 
+The restarting facilities in Akka Typed do not work with untyped children. 
```

Comment:
Same question here, should we change the default when Untyped started from Typed?